Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -42,6 +42,10 @@ public class PressableButton : MonoBehaviour, IMixedRealityTouchHandler
         [Tooltip("Speed of the object movement on release.")]
         private float returnRate = 25.0f;
 
+        [SerializeField]
+        [Tooltip("Ensures that the button can only be pushed from the front. Touching the button from the back or side is prevented.")]
+        private bool enforceForwardPush = true;
```

Comment:
I'm curious to hear if anybody has an idea for a more descriptive name here. What do you think?

Reply:
Seems 100% clear to me, especially with the tooltip.